pages/linux/libreoffice.md

```diff
@@ -0,0 +1,20 @@
+# libreoffice
+
+> Use the powerful and free office suite, LibreOffice, from the command line.
+> More information: <https://www.libreoffice.org/>.
+
+- Open the given files in viewer mode (read-only):
```

Comment:
I think `read-only` has a well know meaning. And we define here the condition of `space-separated list of files` doesn't need to be mention in the following examples.
```suggestion
- Open a space-separated list of files in read-only mode:
```